pkg/arvo/tests/sys/zuse/format.hoon

```diff
@@ -0,0 +1,597 @@
+/+  *test
+=,  format
+|%
+:: split a cord on newlines
```

Comment:
Here and throughout, comments should be preceded by two spaces, and followed by a blank comment line:

```
::  split a cord on new lines
::
```